### DISCUSSION TITLE
How we can show form errors below the respective form fields in jinja template

### DISCUSSION THREAD

**Discussion by agnihotri7** [fastapi/fastapi#8364] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@app.put("/profile", dependencies=[Depends(login_required)])
async def profile(request: Request):
    data_body = await request.form()
    form = ProfileUpdateForm(request)
    await form.load_data()
    if form.is_valid():
        return templates.TemplateResponse(
            "profile.html", form.__dict__
        )
    return templates.TemplateResponse(
        "profile.html", form.__dict__
    )



profile.html
<body>
{% for error in errors %}
  <p>{{error}}</p>
{% end %}

<input name='email' type='email' value={{email}}>

</body>
```

### Description

Similar to django form functionality, I want to access each field errors separately in the template so that i can show the errors just below each field. Something like if form.field.erros then show the errors. 

### Operating System

Linux

### Operating System Details

v 20.06

### FastAPI Version

0.79.0

### Python Version

3.9

**Comment by iudeen** [comment]
Not sure if I understood your question correctly...

Do you want to validate your request? If yes, you can use Pydantic model to do so.

**Comment by agnihotri7** [comment]
@iudeen 
I am creating a webapp in fastAPI.
After Pydentic model validations we pass all the errors in template and show them in template as i mentioned in above code.

The problem here is that we are showing all the errors in template at once but I want to show the error message just below the respective field. 

[See the screenshot attached.](https://i.imgur.com/qeQrNOx.png)
